src/lib/configs/project/workers.ts

```diff
@@ -72,6 +72,7 @@ export type Deal = WorkerInfo & {
   dealIdOriginal: string;
   chainNetworkId: number;
   chainNetwork?: ChainENV;
+  matched?: boolean;
```

Comment:
Same. Let's take the info from chain.

Reply:
yeah this is the exact same value you commented on previously. Would be great if it's possible to take this from chain